package.json

```diff
@@ -24,7 +24,13 @@
     "prepare-release": "./prepare-release.sh",
     "release": "./release.sh",
     "prune-arc-build": "node ./pruneArcBuild.js",
-    "postinstall": "npm run prune-arc-build"
+    "postinstall": "npm run prune-arc-build",
+    "verify.initialize": "cd ./node_modules/@daostack/arc && npm install openzeppelin-solidity && cd ../infra && npm install openzeppelin-solidity",
```

Comment:
perhaps `npm explore @doastack/arc -- npm install openzeppelin-solidity` is a bit more robust.

Reply:
Good idea, forgot about that ability

Reply:
done